src/main/java/Application.java

```diff
@@ -0,0 +1,39 @@
+import controller.CarNameException;
+import controller.RacingCarController;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+import model.RacingCar;
+import view.RacingCarView;
+import view.TryCountException;
+
+public class Application {
+
+    public static void main(String[] args) throws CarNameException, TryCountException {
+        RacingCarView racingCarView = new RacingCarView();
+        RacingCarController racingCarController = new RacingCarController();
+
+        racingCarView.inputCarName();
+
+        String[] carNames = racingCarView.getCarName();
+        racingCarController.checkCarNamesValid(carNames);
+        List<String> carNamesList = Arrays.asList(carNames);
+        Stream<String> carNamesStream = carNamesList.stream();
+
+        racingCarView.inputTryCount();
+        int tryCount = racingCarView.getTryCount();
+
+        ArrayList<RacingCar> racingCars = new ArrayList<>();
+        carNamesStream.forEach(carName -> racingCars.add(new RacingCar(carName)));
+
+        racingCarView.printRacingCarsCurrentLocationByEachCounter(tryCount, racingCars);
+        int maxValue = racingCarController.findMaxValueOfRacingCarsLocation(racingCars);
+
+        for (RacingCar racingCar : racingCars) {
+            racingCar.checkWinner(maxValue);
+        }
+        racingCarView.finalWinner(racingCars);
```

Comment:
```suggestion
        racingCarView.showFinalWinner(racingCars);
```
변수나 메소드명을 정할 때 컨벤션을 참고하여 스스로의 규칙을 정해서 네이밍하면 좋겠습니다.